src/hub/Hub.sol

```diff
@@ -267,14 +260,7 @@ contract Hub is Multicall, Auth, Recoverable, IHub, IHubGatewayHandler {
         (uint128 payoutAssetAmount,) =
             shareClassManager.revokeShares(unlockedPoolId, scId, payoutAssetId, navPerShare, valuation);
 
-        uint128 valueChange = holdings.decrease(unlockedPoolId, scId, payoutAssetId, valuation, payoutAssetAmount);
-
-        accounting.addCredit(
-            holdings.accountId(unlockedPoolId, scId, payoutAssetId, uint8(AccountType.Asset)), valueChange
-        );
-        accounting.addDebit(
-            holdings.accountId(unlockedPoolId, scId, payoutAssetId, uint8(AccountType.Equity)), valueChange
-        );
+        sender.sendRevokedShares(unlockedPoolId, scId, payoutAssetId, payoutAssetAmount);
```

Comment:
Holding update is now triggered when a user claims a redemption on CV side inside `AsyncRequests.withdraw`(see `AsyncRequests._withdraw`).